servers/src/main/java/tachyon/worker/block/BlockDataManager.java

```diff
@@ -430,6 +430,27 @@ public void userHeartbeat(long userId, List<Long> metrics) {
   }
 
   /**
+   * Gets the metadata of a block given its blockId or throws IOException.
+   *
+   * @param blockId the block ID
+   * @return metadata of the block or null
+   * @throws IOException if no BlockMeta for this blockId is found
+   */
+  public BlockMeta getBlockMeta(long blockId) throws IOException {
```

Comment:
Could you rename this to something like `getVolatileBlockMeta`? @apc999 do you have any preferences for the naming convention?